src/plugins/vuetify.js

```diff
@@ -1,31 +1,27 @@
 import Vue from 'vue';
 import Vuetify from 'vuetify/lib';
-import { preset } from 'vue-cli-plugin-vuetify-preset-rally/preset';
 import '@mdi/font/css/materialdesignicons.css'; // for the default material design icons
 import 'material-design-icons-iconfont/dist/material-design-icons.css';
+import { preset } from 'vue-cli-plugin-vuetify-preset-reply/preset';
 
 Vue.use(Vuetify);
-
 export default new Vuetify({
   preset,
   theme: {
+    // default: 'dark',
     options: {
       customProperties: true,
     },
-    icons: {
-      iconfont: 'mdi', // default - only for display purposes
-      // iconfont: 'md', // default - only for display purposes
-    },
     themes: {
-      light: {
-        primary: '#ee44aa',
-        secondary: '#424242',
-        accent: '#82B1FF',
-        error: '#FF5252',
-        info: '#2196F3',
-        success: '#4CAF50',
-        warning: '#FFC107',
-      },
+      // light: {
```

Comment:
these comments are the values for the custom theme I created. We may not want to go there, and instead use the Material samples from the Material team.